.changeset/afraid-dogs-brake.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Shouldn't we add a changeset about these changes?

Reply:
I was thinking to avoid adding one as we never really released the Coinbase provider, so in some way we revert the changes. But sure let me add one